api/src/main/resources/public/api/v1/jobs.raml

```diff
@@ -276,7 +276,7 @@ post:
     #
     get:
       description:
-        Get the list of all runs for this jobId.
+        Get the list currently active runs for this jobId. Once the run is finished, it will not be available through this endpoint.
```

Comment:
`Get the list currently active` =>  `Get the list of currently active`

Reply:
agreed

Reply:
thanks!